pt/lessons/basics/enum.md

```diff
@@ -119,30 +140,35 @@ iex> Enum.max([], fn -> :bar end)
 
 ### reduce
 
-Com `reduce` podemos transformar nossa coleção em um único valor, para fazer isto aplicamos um acumulador opcional (`10` por exemplo) que será passado a nossa função; se não prover um acumulador, o primeiro valor será usado:
+Com `reduce/3` podemos transformar nossa coleção em um único valor, para fazer isto aplicamos um acumulador opcional (`10` neste exemplo) que será passado a nossa função; se não prover um acumulador, o primeiro valor será usado:
 
 ```elixir
 iex> Enum.reduce([1, 2, 3], 10, fn(x, acc) -> x + acc end)
 16
+
 iex> Enum.reduce([1, 2, 3], fn(x, acc) -> x + acc end)
 6
+
 iex> Enum.reduce(["a","b","c"], "1", fn(x,acc)-> x <> acc end)
 "cba1"
 ```
 
 ### sort
 
-Ordenar nossas coleções é fácil com uma das funções de ordenação. A primeira opção disponível utiliza o critério de ordenação de termos do Elixir para determinar a ordem da ordenação:
+Ordenar nossas coleções é fácil com uma das funções de ordenação.
+
+`sort/1` usa a funcionalidade de ordenação do Erlang para determinar a ordem:
 
 ```elixir
 iex> Enum.sort([5, 6, 1, 3, -1, 4])
 [-1, 1, 3, 4, 5, 6]
 
 iex> Enum.sort([:foo, "bar", Enum, -1, 4])
 [-1, 4, Enum, :foo, "bar"]
+```, Enum, :foo, "bar"]
```

Comment:
Comparing to the English version, I believe this line was added unintentionally.